inventory/group_vars/all.yml

```diff
@@ -50,18 +50,15 @@ swapfile_size: false
 
 #----------------------------------------------------------------------
 # Rails variables
-ruby_version: 3.0.3
+ruby_version: 3.1.4
 
 rbenv_extra_depends:
   - libjemalloc-dev
 
 ruby_compile_options: "--with-jemalloc"
 
 ruby_versions:
-  - version: 2.7.3
-    env:
-      RUBY_CONFIGURE_OPTS: "{{ ruby_compile_options }}"
-  - version: 3.0.3
+  - version: 3.1.4
     env:
       RUBY_CONFIGURE_OPTS: "{{ ruby_compile_options }}"
```

Comment:
Ah, good one. New Ruby versions are now automatically installed during deploys. So we don't need to run Ansible for that. This is just for the initial version.

I wish that we could skip the version and take the version number from the ofn code but this role requires a version. So if we want to set up rbenv without installing ruby straight away then we can't use this role. Just one of the annoying Ansible things that we could replace with shell scripts...